workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -6,8 +6,8 @@ description: |-
   Taskcluster Task definition.
 type: object
 required:
-- command
-- maxRunTime
+  - command
+  - maxRunTime
```

Comment:
I suspect this doc was already formatted, but perhaps using a different indentation strategy (non-indented list items).

We can switch, but we should probably decide on a standard so that we don't switch back and forth. Perhaps we can do that in a different PR to reduce the noise here, and add a linting setting at the same time to keep the same formatting standard?